src/logic/function/basic.lean

```diff
@@ -348,6 +348,36 @@ by {funext b, by_cases b = a; simp [update, h]}
 
 end update
 
+namespace injective
+
+noncomputable theory
+local attribute [instance, priority 10] classical.prop_decidable
+
+variables {α β γ : Type*} {f : α → β} (hf : injective f)
```

Comment:
```suggestion
variables {α β γ : Type*} {f : α → β}
```
You repeat this assumption each time anyway.

Reply:
I've removed the repeated assumptions :grinning: